_docs/operations/encryption.md

```diff
@@ -0,0 +1,108 @@
+PVCs or you can directly create volumes using the StorageOS CLI or GUI with the
+encryption label applied. 
+
+For more in depth discussion of how encryption works please see the [Encryption
+concepts](/docs/concepts/encryption) page.
+
+## Required labels
+
+The `storageos.com/encryption` label must be applied to the volume when it is
+created. The encryption status of a volume cannot be changed after a volume has been
+created.
+
+You can pass the label using a PVC
+
+```yaml
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: pvc0002
+  labels:
+    "storageos.com/encryption": "true"
+  annotations:
+    volume.beta.kubernetes.io/storage-class: fast
+spec:
+  accessModes:
+    - ReadWriteOnce
+  resources:
+    requests:
+      storage: 5Gi
+```
+
+You can also pass the encryption label when creating volumes using the CLI
+```bash
+$ storageos volume create encrypted --label storageos.com/encryption=true
+```
+
+You can also add the encryption label when creating a volume with the GUI
+
+## Backing up Secrets
+
+The keys that are used to encrypt a volume are stored in a Kubernetes secret.
+StorageOS does not have access to the keys that are used to encrypt a volume,
+as such if the keys are lost the volume **cannot** be decrypted.
+
+As a precautionary measure we recommend that you backup the Kubernetes secrets
+used to store the encryption keys.
+
+StorageOS will create one secret per encrypted volume and the secrets are
+created in whatever namespace StorageOS is installed into. By default, this is
+the StorageOS namespace.
+```bash
+$ kubectl get secrets -n storageos
+NAME                                           TYPE                                  DATA   AGE
+default-token-t4xl4                            kubernetes.io/service-account-token   3      20h
+init-secret                                    Opaque                                2      20h
+ns-key.default                                 Opaque                                1      20h
+storageos-daemonset-sa-token-qs7r6             kubernetes.io/service-account-token   3      20h
+vol-key.4276fc07-7d85-70bf-35a0-f0b005e55e0f   Opaque                                4      1m
+```
+In the output above there is a ns-key.default and a vol-key. A ns-key is
+created for each [StorageOS namespace](/docs/concepts/namespaces) in the format
+ns-key.{namespace}. A vol-key is created for every encrypted volume. The
+vol-keys are named vol-key.{volume-id}. The volume id can be retrieved by
+inspecting the volume.
+
+```bash
+$ kubectl get pvc --show-labels
+NAME      STATUS   VOLUME                                     STORAGECLASS   AGE   LABELS
+pvc-1     Bound    pvc-46c16185-40dd-11e9-91ad-0a57700a78b4   fast           9m    <none>
+pvc0002   Bound    pvc-1c68f013-40dd-11e9-91ad-0a57700a78b4   fast           10m   storageos.com/encryption=true
+$ storageos v inspect default/pvc-1c68f013-40dd-11e9-91ad-0a57700a78b4 | grep -m1 id
```

Comment:
Add a comment to explain what are you doing. 

```bash

# Find the id of the volume
$ storageos volume inspect ...

# Find the secret related to that volume
$ kubectl get secret ...
```

Please don't use shortcuts in the docs such as `storageos v i` instead of `storageos volume inspect` as it can be confusing.